plugin.audio.intergalacticfm/main.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+# Module: default
+# Author: Pander
+# Created on: 2019-03-04
+# License: GPL v.3 https://www.gnu.org/copyleft/gpl.html
+
+from json import load
+from sys import argv
+from os.path import isfile
+import requests
+
+from urllib import urlencode
+from urlparse import parse_qsl
+
+
+import xbmcgui
+import xbmcplugin
+import xbmcaddon
+from xbmc import log, LOGNOTICE, LOGERROR
```

Comment:
You are not using `LOGERROR`, so it can be removed.

Reply:
done